src/components/UserMounts/UserMountsTable.js

```diff
@@ -7,23 +7,24 @@ import {
   TableHead,
   TableRow,
   Paper,
-} from "@material-ui/core";
-import CheckIcon from "@material-ui/icons/Check";
-import ClearIcon from "@material-ui/icons/Clear";
-import { USER_COLUMN } from "utils/constants";
-import { EditUserMountsButton } from "components/UserMounts/EditUserMounts/EditUserMountsButton";
-import PropTypes from "prop-types";
-import "components/Table.scss";
+} from '@material-ui/core';
+import CheckIcon from '@material-ui/icons/Check';
+import ClearIcon from '@material-ui/icons/Clear';
+import { USER_COLUMN } from 'utils/constants';
+import { EditUserMountsButton } from 'components/UserMounts/EditUserMounts/EditUserMountsButton';
+import { TableCellWrapper } from 'components/TableCellWrapper';
+import PropTypes from 'prop-types';
+import 'components/Table.scss';
 
 export const UserMountsTable = (props) => {
   return (
     <TableContainer component={Paper}>
-      <Table className="table" aria-label="customized table">
-        <TableHead className="table-head">
+      <Table className='table' aria-label='customized table'>
```

Comment:
Go over the styles and remove classes that don't exist anymore

Reply:
extraneous styles and classes eliminated